docs/siem/detections/api/exceptions/api-create-exception-item.asciidoc

```diff
@@ -0,0 +1,372 @@
+[[exceptions-api-create-exception-item]]
+=== Create exception item
+
+Creates an exception item and associates it with the specified
+<<exceptions-api-create-container, exception container>>.
+
+See <<lists-api-overview>> for information about creating exception items from
+lists, such as a list of IP addresses or host names.
+
+NOTE: Before creating exception items, you must create an exception container.
+
+[IMPORTANT]
+==============
+Endpoint rule exception items cannot use
+<<lists-api-overview, lists>> (the `list` in the `entries` array), and the
+following fields cannot be used in exception queries (as `field` values in the
+`entries` object):
+
+* `file.Ext.quarantine_path`
+* `file.Ext.quarantine_result`
+* `process.entity_id`
+* `process.parent.entity_id`
+* `process.ancestry`
+==============
+
+==== Request URL
+
+`POST <kibana host>:<port>/api/exception_lists/items`
+
+==== Request body
+
+A JSON object with these fields:
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`comments` |comments[] a|Array of `comment` fields:
+
+* `comment` (string): Comments about the exception item.
+
+|No, defaults to empty array.
+
+|`description` |String |Describes the exception item. |Yes
+|`entries` |<<entries-object-schema, entries[]>> |Array containing the
+exception queries. Boolean `AND` logic is used to evaluate the relationship
+between array elements. If you want to use `OR` logic, create a separate
+exception item. |Yes
+|`list_id` |String |ID of the associated <<exceptions-api-create-container, exception container>>. |Yes
+|`item_id` |String |Unique identifier of the exception item. |No, automatically
+created when it is not provided.
+|`meta` |Object |Placeholder for metadata about the exception item. |No
+|`name` |String |The exception item's name. |Yes.
+|`namespace_type` |String a|Determines whether the exception item is available
+in all {kib} spaces or just the space in which it is created, where:
+
+* `single`: Only available in the {kib} space in which it is created.
+* `agnostic`: Available in all {kib} spaces.
+
+|No, defaults to `single`.
+|`tags` |String[] |String array containing words and phrases to help categorize
+exception items. |No 
+|`type` |String a|Exception query type, must be `simple`. |Yes
+|`_tags` |String[] a|For endpoint rules only, defines the OS on which the
```

Comment:
@peluja1012 @madirey @marshallmain @FrankHassanabad is this a field we want to expose to users? Usually the `_` fields are just internal, right? Would we want users to use the `tags` instead as `_tags` is more for us?

Reply:
Good point! Not good if the user is able to override `_tags` ...

Reply:
BUT that's the only way for them to target an OS for endpoint exceptions...

Reply:
Should we have them use `tags` and it gets copied over to `_tags` in the background. Because if we start relying on things to be in `_tags` and the user has access then `_tags` becomes totally unreliable, no?

Reply:
@yctercero I think at minimum, we need some additional validation on `_tags` at the API level... I think 1) requiring either `endpoint` or `detection` as one of the tags, and 2) if `endpoint`, then at least 1 accompanying `os:<os-name>` tag ...

Reply:
It's a separate issue, but if we want to prevent the user from accessing `_tags` directly, it seems like we'd just want to add separate fields for 1) OS and for 2) exception type ... it's a little awkward to have these as part of `_tags` from an API usability standpoint, but it will work a lot better if we add the validation.

Reply:
That makes sense. I like your suggestion of adding those as separate fields. Are we prevented adding that in a next release if we leave it as is for now?

Reply:
It would require an SO migration (which I think we've determined we can do, and will need to do soon anyway) along with code changes. Depending on how we want to hook it up, it may only require 1 new field ... we could, for example, assume that the existence of any OS tag makes it an endpoint exception. We'd need to talk through the different scenarios... @peluja1012 @spong